src/wp-includes/rest-api/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -262,6 +262,12 @@ public function get_items( $request ) {
 			$prepared_args['offset'] = $prepared_args['number'] * ( absint( $request['page'] ) - 1 );
 		}
 
+		$is_head_request = $request->is_method( 'HEAD' );
+		if ( $is_head_request ) {
+			// Force the 'fields' argument. For HEAD requests, only post IDs are required to calculate pagination.
+			$prepared_args['fields'] = 'ids';
```

Comment:
Should we avoid priming comment meta here as well?

Reply:
> Should we avoid priming comment meta here as well?
Yes, I think we should priming comment meta here.
Fixed in 294584e3a25df251df985b2c28525ac264b9adff